_episodes/03-configuration.md

```diff
@@ -184,71 +209,131 @@ rootpath:
+> How does ESMValTool find the data?
+>
+> Note:
+> to get the data, check instruction in
+> [Setup]({{ page.root }}{% link setup.md %}).
+>
+>> ## Solution
+>>
+>> - Are you working on your own local machine?
+>>   You need to add the root path of the folder where the data is available
+>> to the `config-user.yml` file as:
+>>```yaml
+>>  rootpath:
+>>  ...
+>>    CMIP5: ~/esmvaltool_tutorial/data
+>>    obs4mips: ~/esmvaltool_tutorial/data
+>>```
+>>
+>> - Are you working with on a computer cluster like Jasmin or DKRZ?
+>>   Site-specific path to the data are already listed at the end of the
+>> `config-user.yml` file. You need to uncomment the related lines.
+>> For example, on Jasmin:
+>>```yaml
+>>  # Site-specific entries: Jasmin
+>>  # Uncomment the lines below to locate data on JASMIN
+>>  rootpath:
+>>  #  CMIP6: /badc/cmip6/data/CMIP6
+>>    CMIP5: /badc/cmip5/data/cmip5/output1
+>>  #  CMIP3: /badc/cmip3_drs/data/cmip3/output
+>>  #  OBS: /group_workspaces/jasmin4/esmeval/obsdata-v2
+>>  #  OBS6: /group_workspaces/jasmin4/esmeval/obsdata-v2
+>>    obs4mips: /group_workspaces/jasmin4/esmeval/obsdata-v2
+>>  #  ana4mips: /group_workspaces/jasmin4/esmeval/obsdata-v2
+>>  #  CORDEX: /badc/cordex/data/CORDEX/output
+>>```
+>>
+>> - For more information about setting the rootpath, see also the ESMValTool
+>> [documentation](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/quickstart/find_data.html).
+> {: .solution}
+{: .challenge}
 
 ## Directory structure for the data from different projects
 
 Input data can be from various models, observations and reanalysis data that
 adhere to the [CF/CMOR standard](https://cmor.llnl.gov/). The ``drs`` setting
-describes the file structure. Let's use ``default`` for ``CMIP5`` in our example
-here:
+describes the file structure.
 
-```yaml
-drs:
-  CMIP5: default
-```
+The ``drs`` setting describes the file structure for several projects (e.g.
+CMIP6, CMIP5, obs4mips, OBS6, OBS) on several key machines
+(e.g. BADC, CP4CDS, DKRZ, ETHZ, SMHI, BSC). For more
+information about ``drs``, you can visit the ESMValTool documentation on
+[Data Reference Syntax (DRS)](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/quickstart/find_data.html#cmor-drs).
 
-> ## Available drs
+> ## Set the correct drs
 >
-> The ``drs`` setting describes the file structure for several projects (e.g.
-> ``CMIP6``, ``CMIP5``, ``obs4mips``, ``OBS6``, ``OBS``) on several key machines
-> (e.g. ``BADC``, ``CP4CDS``, ``DKRZ``, ``ETHZ``, ``SMHI``, ``BSC``). For more
-> information about ``drs``, you can visit the ESMValTool
-> [documentation](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/quickstart/find_data.html#cmor-drs).
-{: .callout}
-
-> ## Make your own configuration file
+> In this lesson, we will work with data from
+> [CMIP5](https://esgf-node.llnl.gov/projects/cmip5/)
+> and [obs4mips](https://esgf-node.llnl.gov/projects/obs4mips/).
+> How does ESMValTool know the data structure?
```

Comment:
```suggestion
> Set the correct `drs` for the tutorial. If you're working on your local machine, use the 'default'. Otherwise, find the correct drs specification for your cluster.
```

Reply:
You are giving the answer to the challenge in this line. I revised the question a bit without mentioning the answer.